http/headers/content-security-policy.json

```diff
@@ -1302,6 +1356,48 @@
                     "version_added": "7.1"
                   }
                 },
+              "With external scripts": {
```

Comment:
Can you make this a key like "external_scripts" and then add a `desc` that explains what this is in English?

"external_scripts": { "desc": "With external scripts", "support": ... }